### PR TITLE
Update installation.md - Grammar Errors

### PR DESCRIPTION
**Pull request by midhunbiju**
I fixed the Grammar Errors

This is my commit message

Signed-off-by: midhunbiju <midhunbijutvm@gmail.com>

### CODE REVIEW

**Comment by sonasingh46** [comment]
@midhunbiju 
Please fix the DCO sign.

**Comment by midhunbiju** [comment]
@sonasingh46 how can i fix DCO sign.
Pls help me

File: docs/installation.md [hunk 1]
```diff
@@ -47,16 +47,16 @@ kubectl -n kube-system patch deploy/tiller-deploy -p '{"spec": {"template": {"sp
 
 ### Install OpenEBS using OpenEBS Helm Charts
 
-You have now set up helm and RBAC by following above [step](/docs/next/installation.html#setup-helm-rbac). Next,You should clone the latest OpenEBS repository. If you have cloned OpenEBS repository already, verify that it is updated.
+You have now set up helm and RBAC by following above [step](/docs/next/installation.html#setup-helm-rbac). Next, You should clone the latest OpenEBS repository. If you have cloned OpenEBS repository already, verify that it is updated.
```
Comment (muratkars) [inline docs/installation.md]:
```suggestion
You have now set up Helm and RBAC by following above [step](/docs/next/installation.html#setup-helm-rbac). Next, you should clone the latest OpenEBS repository. If you have cloned OpenEBS repository already, verify that it is updated.
```

File: docs/installation.md [hunk 2]
```diff
@@ -72,7 +72,7 @@ helm repo add openebs-charts https://openebs.github.io/charts/
 helm repo update
 ```
 
-Now you are ready to install OpenEBS using helm using following command. OpenEBS will install in "**default**" namespace.
+Now you are ready to install OpenEBS using helm using the following command. OpenEBS will install in "**default**" namespace.
```
Comment (muratkars) [inline docs/installation.md]:
```suggestion
Now you are ready to install OpenEBS using Helm using the following command. OpenEBS will install in "**default**" namespace.
```

File: docs/installation.md [hunk 3]
```diff
@@ -152,7 +152,7 @@ You can install OpenEBS cluster by running the following command.
 kubectl apply -f https://openebs.github.io/charts/openebs-operator-0.7.0.yaml
 ```
 
-OpenEBS control plane pods are created under “**openebs**” namespace. CAS Template,default Storage Pool and default Storage Classes are created after executing the above command.Now select your storage to provision OpenEBS volume from [here](/docs/next/installation.html#select-your-storage-engine).
+OpenEBS control plane pods are created under “**openebs**” namespace. CAS Template, default Storage Pool and default Storage Classes are created after executing the above command.Now select your storage to provision OpenEBS volume from [here](/docs/next/installation.html#select-your-storage-engine).
```
Comment (muratkars) [inline docs/installation.md]:
```suggestion
OpenEBS control plane pods are created under “**openebs**” namespace. CAS Template, default Storage Pool and default Storage Classes are created after executing the above command. Now select your storage to provision OpenEBS volume from [here](/docs/next/installation.html#select-your-storage-engine).
```

**Comment by ranjithwingrider** [comment]
There was a changes in the way of installation from the latest release. Thank you for your effort though.

**Pull request closed without merging**